lib/models/src/main/java/org/sagebionetworks/repo/model/dao/discussion/DiscussionThreadDAO.java

```diff
@@ -181,4 +184,41 @@ public DiscussionThreadBundle createThread(String forumId, String threadId,
 	 */
 	public boolean isThreadDeleted(String threadId);
 
+	/**
+	 * 
+	 * @param entityId
+	 * @param limit
+	 * @param offset
+	 * @param order
+	 * @param ascending
+	 * @param filter
+	 * @param projectIds
+	 * @return a paginated list of threads that mentioned the enityId
```

Comment:
@return a paginated list of threads that are in the given projectIds and referenced the given enityId